src/services/lnd/index.ts

```diff
@@ -404,6 +388,30 @@ export const LndService = (): ILightningService | LightningServiceError => {
     }
   }
 
+  const listInvoices = async (
+    lnd: AuthenticatedLnd,
+  ): Promise<LnInvoiceLookup[] | LightningServiceError> => {
+    try {
+      let after: PagingStartToken | PagingContinueToken | PagingStopToken = undefined
+      let rawInvoices = [] as GetInvoicesResult["invoices"]
+      while (after !== false) {
+        const pagingArgs: {
+          token?: PagingStartToken | PagingContinueToken
+        } = after ? { token: after } : {}
+        const { invoices, next } = await getInvoices({ lnd, ...pagingArgs })
+        rawInvoices = [...rawInvoices, ...invoices]
+        after = (next as PagingContinueToken) || false
+      }
+      return rawInvoices.map(translateLnInvoiceLookup)
+    } catch (err) {
+      const errDetails = parseLndErrorDetails(err)
+      switch (errDetails) {
+        default:
```

Comment:
are 3 lines are useful in any way?

Reply:
I did this as a consistency thing to signal to anybody else maintaining these. All the `catch` blocks in the lightning service have this pattern so that we can easily add known lnd error details as new cases and handle them in the switch as we discover them.

For some catch blocks, we haven't come across specific lnd errors we'd like to handle as yet, so their switches only have the default switch case.